include/NovelRT.Interop/Input/NrtInteractionService.h

```diff
@@ -4,32 +4,24 @@
 #ifndef NOVELRT_INTEROP_INPUT_INTERACTIONSERVICE_H
 #define NOVELRT_INTEROP_INPUT_INTERACTIONSERVICE_H
 
-#include "../Maths/NrtGeoVector2F.h"
-#include "../NrtInteropUtils.h"
-#include "../NrtTransform.h"
-#include "../Windowing/NrtWindowingService.h"
-#include "NrtBasicInteractionRect.h"
-#include "NrtKeyCode.h"
-#include "NrtKeyStateFrameChangeLog.h"
```

Comment:
How come these are suddenly not needed? This seems wrong to me.

Reply:
Because the enums and opaque handles were moved to a namespace specific `Nrt*Typedefs.h` folder, which helps with generating bindings and simplifies the file dependency chain.